### PR TITLE
[IMP] mail: introduce ThreadViewTopbar/onClickPhone

### PR DESCRIPTION
**Pull request by alexkuhn**
Task-2579306

### CODE REVIEW

**Review by oomsveta — COMMENTED** [review]
Didn't you forget to update the handler in the template?

**Comment by alexkuhn** [comment]
> Didn't you forget to update the handler in the template?

Oops. Changes done!

File: addons/mail/static/src/components/thread_view_topbar/thread_view_topbar.xml [hunk 1]
```diff
@@ -50,7 +50,7 @@
                         <button class="o_ThreadViewTopbar_unstarAllButton btn btn-secondary" t-att-disabled="threadViewTopbar.threadView.messages.length === 0" t-on-click="threadViewTopbar.onClickUnstarAll">Unstar all</button>
                     </t>
                     <t t-if="threadViewTopbar.thread and threadViewTopbar.thread.hasCallFeature and threadViewTopbar.thread.rtcSessions.length === 0">
-                        <button class="o_ThreadViewTopbar_callButton o_ThreadViewTopbar_button o-active" t-att-disabled="threadViewTopbar.thread.hasPendingRtcRequest" title="Start a Call" t-on-click="_onClickPhone">
+                        <button class="o_ThreadViewTopbar_callButton o_ThreadViewTopbar_button o-active" t-att-disabled="threadViewTopbar.thread.hasPendingRtcRequest" title="Start a Call" t-on-click="threadviewTopbar.onClickPhone">
```
Comment (oomsveta) [inline addons/mail/static/src/components/thread_view_topbar/thread_view_topbar.xml:53]: `threadviewTopbar` -> `threadViewTopbar`?

**Review by oomsveta — APPROVED** [review]

**Comment by alexkuhn** [comment]
Move to grouped PR: https://github.com/odoo/odoo/pull/89076

**Pull request closed without merging**